src/libprojectM/MilkdropPreset/Shaders/PresetWarpVertexShaderGlsl330.vert

```diff
@@ -16,8 +16,8 @@ precision mediump float;
 layout(location = 0) in vec2 vertex_position;
 layout(location = 1) in vec2 rad_ang;
 layout(location = 2) in vec4 transforms;
-layout(location = 3) in vec2 center;
-layout(location = 4) in vec2 distance;
+layout(location = 3) in vec2 center2;
```

Comment:
Can you please explain why changing this fixes it?

Reply:
Same question. The only explanation I could come up with is that Android's shader Compiler somehow has `center` and `distance` as keywords.

I'd not postfix those variables with a 2 in the case, as there's clearly no 1 here. We could instead rather prefix the variables with `in_` or something, which would both fix the potential keyword issue and not make the names look weird.

Reply:
I guess renaming the distance variable should suffice, as I cannot find any keyword or function `center` in any GL standard.

Could you try that, keeping `center` as-is and for example rename `distance` to `warpDistance`?